docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,181 @@
+    ```bash
+    git checkout master
+    git remote add demo https://github.com/almos98/gcp-filestore-csi-driver
+    ```
+2. Apply changes to personal fork.
+    ```bash
+    git fetch demo refs/heads/windows_demo
+    git checkout -b csi_prototype
+    COMMIT=$(git ls-remote demo | grep refs/heads/windows_demo | cut -f 1)
+    git cherry-pick $COMMIT
+    ```
+3. Build Windows binary and upload it to the bucket.
+    ```bash
+    make windows-local
+    gsutil cp bin/gcfs-csi-driver.exe gs://$PROJECT-bucket/
+    ```
+## Preparing the cluster
+1. Go back to your k/k workspace where the patched kubernetes binaries were built and bring up a cluster using those binaries.
+    
+    **WARNING:** This wil tear down an existing e2e cluster in your project.
+    ```bash
+    PROJECT=${CLOUDSDK_CORE_PROJECT} \
+    NUM_NODES=2 \
+    NUM_WINDOWS_NODES=3 \
+    KUBE_GCE_ENABLE_IP_ALIASES=true \
+    KUBERNETES_NODE_PLATFORM=windows \
+    LOGGING_STACKDRIVER_RESOURCE_TYPES=new \
+    KUBERNETES_SKIP_CONFIRM=y \
+    go run ./hack/e2e.go -- --up
+    ```
+
+2. From Filestore driver workspace, set up cluster.
+    ```bash
+    ./deploy/kubernetes/cluster_setup.sh
+    ```
+
+3. On one of the Windows nodes, create an SMB share.
+    
+    Remember the name of this node, will be needed next step.
+    ```Powershell
+    $Password = ConvertTo-SecureString "<your password>" -AsPlainText -Force
+    New-LocalUser -Name smbuser -AccountNeverExpires -Password $Password
+    Add-LocalGroupMember -Group "Remote Desktop Users" -Member smbuser
+    New-Item -ItemType "directory" -Path "C:\SMBShare"
+    New-SmbShare -Name "SMBShare" -Path "C:\SMBShare" -FullAccess "smbuser"
+    ```
+
+4. Update necessary YAML files in Filestore driver workspace.
+    
+    Information you need:
+    - Project name
+    - Name of the node that hosts the SMB share
+    - Password from previous step
+    - Name of the node you will be deploying to.
+
+    List of files:
+    - `deploy/kubernetes/manifests/node_windows.yaml`
+    - `examples/kubernetes/windows/secrets.yaml`
+    - `examples/kubernetes/windows/pv.yaml`
+    - `examples/kubernetes/windows/nettest-pod.yaml`
+
+## Deploying
+[//]: # (Use kubelet commands instead of logging in)
+1. Log in to the node you specified in `examples/kubernetes/windows/nettest-pod.yaml`.
+2. Run the filestore driver.
+    ```Powershell
+    $PROJECT = '<your project>'
+    cd C:\
+    gsutil cp gs://$PROJECT-bucket/gcfs-csi-driver.exe .
+    .\gcfs-csi-driver.exe --v=5 --endpoint="unix:/var/lib/kubelet/plugins/filestore.csi.storage.gke.io/csi.sock" --nodeid="(hostname)" --node=true
+    ```
+3. Back from the Filestore driver workspace, run the *daemonset* for the node-driver-registrar container.
+    ```bash
+    ./deploy/kubernetes/driver_start_windows.sh
+    ```
+4. Finally apply all the YAML files.
+    ```bash
+    ./examples/kubernetes/windows/apply.sh
+    ```
+
```

Comment:
Please add a `Demo` section here that shows how you can e.g. exec into the nettest pod, write a file, and then read the file from the SMB server or from another pod.